lib/const.rb

```diff
@@ -215,8 +221,8 @@
 PY_SETUP_INSTALL_OPTIONS_NO_SVEM = "--root=#{CREW_DEST_DIR} --prefix=#{CREW_PREFIX} -O2 --compile"
 PY_SETUP_INSTALL_OPTIONS = "#{PY_SETUP_INSTALL_OPTIONS_NO_SVEM} --single-version-externally-managed"
 
-CREW_FIRST_PACKAGES = %w[libssh curl git pixz shared_mime_info]
-CREW_LAST_PACKAGES = %w[ghc mandb gtk3 gtk4 sommelier]
+CREW_FIRST_PACKAGES = %w[libssh curl git pixz shared_mime_info harfbuzz]
+CREW_LAST_PACKAGES = %w[ghc mandb gtk3 gtk4 sommelier freetype]
```

Comment:
The last package should be sommelier to prevent the postinstall instructions for scrolling past the screen.

Reply:
Do we even need this now with the new dependency resolver?

Reply:
I haven't tested it yet, but I will say that harfbuzz needs to be after freetype, otherwise the more limited freetype files get left on the system... I don't see how the new resolver fixes that.  @supechicken?

Reply:
I remember that this is a part of mitigations and I have completely removed it. Since the post-install part will run after all package's installation now, we can already ensure that the user won't miss any information

Reply:
Looks like it is still in crew @supechicken ?

```
    # Adjust package install ordering for upgrades.
    CREW_FIRST_PACKAGES.each do |pkg|
      # Add package to beginning.
      toBeUpdated.insert(0,toBeUpdated.delete(pkg)) if toBeUpdated.include? pkg
    end
    CREW_LAST_PACKAGES.each do |pkg|
      if toBeUpdated.include? pkg
        # Add package to beginning.
        toBeUpdated.insert(0,toBeUpdated.delete(pkg))
        # Now rotate first package to last package.
        toBeUpdated=toBeUpdated.rotate(1)
      end
    end
```

Reply:
It is in `def upgrade`, which I forgot to remove😅

Reply:
Go ahead and modify this PR to make those changes?

Reply:
Done